cpu/nrf52/radio/nrf802154/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menuconfig KCONFIG_MODULE_NRF802154
+    bool    "Configure nRF802154"
+    depends on MODULE_NRF802154
+    help
+        Configure nRF802154 module using Kconfig.
+
+if KCONFIG_MODULE_NRF802154
+
+config NRF802154_CCA_THRESH_DEFAULT
+    int "Default CCA threshold value"
+    default -90
+    range -127 127
```

Comment:
Range of a `int8_t` is **-128** ... 127, not?

Reply:
oh! My fault. Actually this value is not int8 anymore

Reply:
done. Changed to HEX